src/corefx/System.Globalization.Native/collation.cpp

```diff
@@ -407,6 +401,24 @@ const UCollator* GetCollatorFromSortHandle(SortHandle* pSortHandle, int32_t opti
     return pCollator;
 }
 
+extern "C" int32_t GlobalizationNative_GetSortVersion(SortHandle* pSortHandle)
```

Comment:
Is there any perf implication to this change? i.e. have we made something that needed to be cheap much more expensive?  Is there caching done higher up?